README.md

```diff
@@ -1 +1,48 @@
-# vip-inheritance
\ No newline at end of file
+# VIP inheritance annotation
+## GenemapConverter
+A tool to converts the omim 'genemap2.txt' to a tab separated file that can be used by the VEP Inheritance Plugin.
+It creates a line for all gene symbols in the input file.
+###Usage
+```
+usage: java -jar genemap-mapper.jar -i <arg> -c <arg> [-o <arg>] [-f]
+       [-l] [-p] [-d]
+ -i,--input <arg>    Input genemap2.txt location.
+ -o,--output <arg>   Output TSV file
+ -f,--force          Override the output file if it already exists.
+ -d,--debug          Enable debug mode (additional logging).
+
+usage: java -jar genemap-mapper.jar -v
+ -v,--version   Print version.
+```
+
+###Mapping
+|OMIM Inheritance|Annotation|
```

Comment:
A lot of these values will never occur in CGD I believe.

Reply:
Do you know if there is a standard inheritance mode system (similar to the variant classification system)? Mapping to standard inheritance modes would allow to replace OMIM with CGD without breaking the pipeline downstream.

Reply:
Offline discussion: we are not aware of such a standard.